src/pages/experimental/edit-pattern/index.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Edit Pattern
+---
+
+<PageDescription>
+
+Editing is making changes to a resource that already exists in the system.
+Updating and editing should use the same behavior as when the resource is created. This should only vary in situations where the number of editable fields is significantly smaller or larger than during creation, or the context of current settings is required.
+
+
+</PageDescription>
+
+#### Status:
+
+[Experimental](/experimental/overview/)
+
+#### Maintainers:
+
+[Vikki Paterson](https://github.com/vikkipaterson), [Chloe Poulter](https://github.com/chloepoulterdesign)
+
+<AnchorLinks>
+
+  <AnchorLink>Low-impact edit</AnchorLink>
+  <AnchorLink>High-impact edit</AnchorLink>
+  <AnchorLink>Optional extras</AnchorLink>
+
+</AnchorLinks>
+
+![Contextual image of an edit modal](images/01.png)
+
+<Caption>Example of an edit modal in context</Caption>
+
+## Low-impact edit
+
+For low-impact edits, allow editing with no warning. This approach is typically used when editing doesn’t have a wider impact.
+
+<Row>
+<Column colLg={8}>
+
+![Example of an edit button in a table](images/02.png)
+
+<Caption>Example of an edit button in a table</Caption>
+
+</Column>
+</Row>
+
+### Modal
+
+If an object was created in a modal, its editable fields can be made available in a modal. A modal can also be used where a small subset of fields are editable.
+If enough fields are editable to require scrolling, use a side-panel or full-page edit dialog instead.
```

Comment:
Did you want this to be two separate paragraphs? It looks like there is a line break, but no empty line between them so it'll render as its one paragraph.

Reply:
One paragraph is probably fine - I imagine it's the ol' trigger happy thumbs again